test/610_proxy_wait_for_weave_test.sh

```diff
@@ -0,0 +1,15 @@
+#! /bin/bash
+
+. ./config.sh
+
+start_suite "Proxy waits for weave to be ready before running container commands"
+weave_on $HOST1 launch-proxy
+BASE_IMAGE=ubuntu
```

Comment:
please use `gliderlabs/alpine` here. It's smaller than ubuntu, has all you need, and is an image we use in other tests.